src/argilla/client/feedback/config.py

```diff
@@ -39,10 +40,11 @@ class DatasetConfig(BaseModel):
     guidelines: Optional[str] = None
 
     def to_yaml(self) -> str:
-        return dump(self.dict())
+        return dump(self.dict({"fields": {"__all__": {"id"}}, "questions": {"__all__": {"id"}}}))
 
     @classmethod
     def from_yaml(cls, yaml: str) -> "DatasetConfig":
+        yaml = re.sub(r"(\n\s*|)id: !!python/object:uuid\.UUID\s+int: \d+", "", yaml)
```

Comment:
this is for backward compatibility with the YAML files that contains the exported IDs, right?